src/main/kotlin/no/nav/syfo/oppfolgingsplanmottak/OppfolgingsplanApi.kt

```diff
@@ -31,13 +34,16 @@ fun Routing.registerOppfolgingsplanApi(
         authenticate(JwtIssuerType.MASKINPORTEN.name) {
             post {
                 val followUpPlanDTO = call.receive<FollowUpPlanDTO>()
+                val uuid = UUID.randomUUID()
+
                 database.storeFollowUpPlan(
-                    uuid = UUID.randomUUID(),
+                    uuid = uuid,
                     followUpPlanDTO = followUpPlanDTO,
-                    organizationNumber = "todo", //Skal hentes fra token claims (supplier ID)
-                    lpsOrgnumber = "todo" //Skal hentes fra token claims (provider ID)
+                    organizationNumber = getOrgnumberFromClaims(),
+                    lpsOrgnumber = getLpsOrgnumberFromClaims()
                 )
-                call.respondText("TODO")
+
+                call.respond(FollowUpPlanResponse(uuid.toString()))
```

Comment:
Er tanken å eksponere intern id i `FollowUpPlanResponse` eller skal det være intern id på sikt?

Reply:
Tanken var at det blir eksternt, men her har du vel noe under arbeid (så er bare å slette / endre på dette)

Var vel også inne på det med å lage egen kolonne for ekstern ID, dersom dette er lurt. Har ikke gjort noe med det enda

Reply:
(Lagde det bare nå for å kunne skrive ferdig en test)